src/schema/v2/ordered_set.ts

```diff
@@ -28,14 +31,52 @@ const OrderedSetType = new GraphQLObjectType<any, ResolverContext>({
     items: {
       type: new GraphQLList(OrderedSetItemType),
       resolve: ({ id, item_type }, _options, { setItemsLoader }) => {
-        return setItemsLoader(id).then(items => {
+        return setItemsLoader(id).then(({ body }) => {
+          const items = body
           return items.map(item => {
             item.item_type = item_type // eslint-disable-line no-param-reassign
             return item
           })
         })
       },
     },
+    itemsConnection: {
+      type: artworkConnection.connectionType,
+      description:
+        "Returns a connection of the items. Only Artwork supported right now.",
+      args: pageable(),
+      resolve: (
+        { id, item_type },
+        { before, after, first, last },
```

Comment:
This can just be `options` (you're desctructuring but then re-structuring it a few lines down)